src/core/gosling.schema.ts

```diff
@@ -155,9 +157,8 @@ export interface CommonViewDef {
 export type Track = SingleTrack | OverlaidTrack | DataTrack | TemplateTrack;
 
 export interface CommonTrackDef extends CommonViewDef {
-    // !! TODO: we can check if the same id is used multiple times.
-    // !! TODO: this should be track-specific and not defined in views.
-    id?: string; // Assigned to `uid` in a HiGlass view config, used for API and caching.
+    /** Assigned to `uid` in a HiGlass view config, used for API and caching. */
+    id?: string;
```

Comment:
```suggestion

```
Since `CommonViewDef` also now has an `id?` property, should this be removed?

Reply:
It can be removed, but I didn't since I wanted to keep the separate comments for the track `id` and the view `id` generated in `schema/gosling.schema.json`.